packages/page-accounts/src/Accounts/Account.tsx

```diff
@@ -472,7 +491,11 @@ function Account ({ account: { address, meta }, className = '', delegation, filt
           )}
         </td>
         <td className='address'>
-          <AddressSmall value={address} />
+          <AddressSmall
+            parentAddress={(typeof meta.parentAddress === 'string') ? meta.parentAddress : undefined}
```

Comment:
```suggestion
            parentAddress={meta.parentAddress || undefined}
```

But even the `|| undefined` should not be needed, these should just be string or undefined (if null, should also be all ok)

Reply:
✅